book_vol_1.txt

```diff
@@ -69,8 +69,8 @@ who is interested can learn and contribute.
 
 About the Author[s]:
 
-        Arash TC is the main author and maintainer of this book. He is currently studying IT in Finland. He's got OSCP/OSCE and a long long long way
-ahead to become a pro. He will appreciate reader's comments and criticisms and contributions. His main interest is low level security and kernel
+        Arash TC is the main author and maintainer of this book. He is currently studying IT in Finland. He's got an OSCP/OSCE and a long long long way

```

Comment:
Yeah I know but there was a slash so I thought you merged them together, I'll fix it with some other stuff and make a pull request tonight...